lib/container_server.go

```diff
@@ -451,6 +451,7 @@ func (c *ContainerServer) LoadSandbox(id string) error {
 		return err
 	}
 
+	// TODO FIXME We currently don't set anything up for the init binary here.
```

Comment:
Do you have a suggestion what needs to be done here? I would suggest to open a follow-up issue instead of leaving `TODO`s in the source code. :innocent: